editor/src/app/sites/site.component.ts

```diff
@@ -40,6 +40,7 @@ import { AppStateModel } from '../app-state/app-state.interface';
           <berta-inline-text-input *ngIf="!modificationDisabled"
                                    [value]="site.name"
                                    (inputFocus)="updateComponentFocus($event)"
+                                   (click)="updateSiteUrl(site.name)"
```

Comment:
It is the wrong event. The original `click` event will be raised every time the `berta-inline-text-input` gets a click, which also includes clicks on buttons inside of it.